go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -162,19 +162,10 @@ func (oc *Controller) updateEgressFirewall(oldEgressFirewall, newEgressFirewall
 	// add the new egressfirewall
 
 	updateErrors = errors.Wrapf(updateErrors, "%v", oc.addEgressFirewall(newEgressFirewall))
-	// delete policy blocking all external traffic
-	stdout, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find",
-		"logical_router_policy", fmt.Sprintf("external-ids:egressFirewall=%s-blockAll", newEgressFirewall.Namespace))
+	// delete rules blocking all external traffic
+	err = deleteEgressFirewallRules(newEgressFirewall.Namespace + "-blockAll")
```

Comment:
this is a great resuse of functions! could you please change the function definition?

`func deleteEgressFirewallRules(namespace string)` error -> `func deleteEgressFirewallRules(externalID string) error` to reflect that you are using the argument as the externalID and change the doc comment to that effect

Reply:
fixed.